web-console/src/links.ts

```diff
@@ -19,7 +19,7 @@
 import hasOwnProp from 'has-own-prop';
 
 // This is set to the latest available version and should be updated to the next version before release
-const DRUID_DOCS_VERSION = '0.20.0';
+const DRUID_DOCS_VERSION = '0.24.0';
```

Comment:
thanks for catching this. we are switching to 24.0.0 starting the next release. can you make that change?

Reply:
@abhishekagarwal87 u r welcome, yes, I'm aware of it, but.. r u sure we need to change the version number to 24.0.0 as well? The version in the pom file is still 0.24.0, fyi, https://github.com/apache/druid/blob/master/pom.xml#L32